src/nightwatch/setup-helper.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const fs = require('fs');
+const loadJsonFile = require('load-json-file');
+const globSync = require('glob').sync;
+
+const interatePackageDirectories = (lernaJSON) => {
+  const srcFolders = [];
```

Comment:
I just retested this, I must have written `*/*-spec.js` initially because my first thought was also to use this filter pattern. Will update.